### PR TITLE
Add lib for resource content delivery repo

### PR DESCRIPTION
**Pull request by simozhan**
Test case is added as well.

### CODE REVIEW

File: lib/pdc/v1/content_delivery_repo.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+module PDC
+  module V1
+    class ContentDeliveryRepo < PDC::Base
+      attributes :arch, :content_category, :content_format,
+                 :name, :product_id, :release_id, :repo_family,
+                 :service, :shadow, :variant_uid
+
```
Comment (ycheng-aa) [inline lib/pdc/v1/content_delivery_repo.rb]: Style/EmptyLinesAroundClassBody: Extra empty line detected at class body end.

File: lib/pdc/version.rb [hunk 2]
```diff
@@ -1,3 +1,3 @@
 module PDC
-  VERSION = '0.1.6'.freeze
+  VERSION = '0.1.7'.freeze
```
Comment (ycheng-aa) [inline lib/pdc/version.rb]: I am not sure if this part should be added in the commit about 'bump version'.

Comment (simozhan) [inline lib/pdc/version.rb]: Ack. Updated the commit message

File: examples/pdc_resource_tests.rb [hunk 3]
```diff
@@ -172,6 +172,19 @@ def token
   end
 end
 
+describe PDC::V1::ContentDeliveryRepo do
+  let(:repo_class) { PDC::V1::ContentDeliveryRepo }
+  let(:existing_repo) { repo_class.all!.first }
+
+  describe '#find' do
```
Comment (sthaha) [inline examples/pdc_resource_tests.rb]: Need a test for this class instead of an example so that we know if something breaks.

File: lib/pdc/version.rb (hunk 2 above)
Comment (sthaha) [inline lib/pdc/version.rb]: should be in a different commit to keep commits atomic as much as possible. 
Alternatively, lets remove bumping up the version since it is part of the release process.

Comment (ycheng-aa) [inline lib/pdc/version.rb]: Sorry for the confusion. As @sthaha said, I meant in a separate commit about 'bump version' in release process.

File: examples/pdc_resource_tests.rb (hunk 3 above)
Comment (simozhan) [inline examples/pdc_resource_tests.rb]: Thanks sthaha. Added. Always instinctively think the file with suffix 'test' is a test file :)

File: spec/pdc/v1/content_delivery_repo_spec.rb [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+describe PDC::V1::ContentDeliveryRepo do
+  before do
+    VCR.insert_cassette fixture_name
+  end
+
+  after do
+    VCR.eject_cassette
+  end
+
+  let(:content_delivery_repos) { PDC::V1::ContentDeliveryRepo.where(release_id: 'ceph-2.1-updates@rhel-7', service: 'pulp') }
```
Comment (sthaha) [inline spec/pdc/v1/content_delivery_repo_spec.rb]: Since this isn't test setup but an actual test, could you please move PDC::V1::ContentDeliveryRepo.where.. inside the actual test?

Please consider also adding the example above as a test

File: lib/pdc/version.rb (hunk 2 above)
Comment (simozhan) [inline lib/pdc/version.rb]: OK. Ack
